resources/overlay/usr/local/bin/fillmem.c

```diff
@@ -55,6 +55,12 @@ int main(int argc, char *const argv[]) {
 
     int pid = fork();
     if (pid == 0) {
+        int oom_fd = open("/proc/self/oom_score_adj", O_WRONLY);
```

Comment:
Nice idea! Could we use `choom` when we call fillmem?  https://man7.org/linux/man-pages/man1/choom.1.html It should be available in the guest.

Reply:
Actually I think if `choom` will work, it would be better. No need to update rootfs with new version of `fillmem`